cmd/connect.go

```diff
@@ -16,6 +17,12 @@ import (
 	"github.com/modal-labs/vprox/lib"
 )
 
+var HEALTH_CHECK_INTERVAL = 3 * time.Second // when we're healthy, how frequently do we
```

Comment:
Go naming conventions are camelCase or CamelCase for constants, and the multiline comment goes before